src/main/java/edu/harvard/iq/dataverse/authorization/Permission.java

```diff
@@ -46,7 +46,7 @@
     EditDataset(BundleUtil.getStringFromBundle("permission.editDataset"), true, Dataset.class),
     ManageDataversePermissions(BundleUtil.getStringFromBundle("permission.managePermissionsDataverse"), true, Dataverse.class),
     ManageDatasetPermissions(BundleUtil.getStringFromBundle("permission.managePermissionsDataset"), true, Dataset.class),
-    ManageFilePermissions(BundleUtil.getStringFromBundle("permission.managePermissionsDatasetFiles"), true, Dataset.class),
+    ManageFilePermissions(BundleUtil.getStringFromBundle("permission.managePermissionsDataFiles"), true, DataFile.class),
```

Comment:
Very Minor change, but let's make this singular to be in parallel with the others.